src/features/utils/index.ts

```diff
@@ -363,14 +364,17 @@ export function generateSaltedHash(password: string): ISaltedHash {
 }
 
 export const convertToMSISDN = (phone: string, countryAlpha3: string) => {
-  const countryCode =
-    callingCountries[countryAlpha3.toUpperCase()].countryCallingCodes[0]
-  if (phone.startsWith(countryCode) || `+${phone}`.startsWith(countryCode)) {
-    return phone.startsWith('+') ? phone : `+${phone}`
+  const data = allCountries.find(
```

Comment:
I don't think this variable is used anywhere so you can safely remove it and the `countryUtils.ts` module